src/capidle/script/isGuessCorrect.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * This function removes whitespace, normalizes upper/lower case,
+ * and removes special characters like macrons.
+ * @param {string} guess
+ */
+const normalizeString = (guess) =>
+  guess
+    .trim()
+    .toLowerCase()
+    .normalize("NFD")
+    .replace(/\p{Diacritic}/gu, "")
+    .replace(/  +/g, " ") // remove multiple spaces
```

Comment:
I am getting a SonarLint error from this line. Do you know if there's a workaround for this? 
![image](https://user-images.githubusercontent.com/78716153/184482640-777bcbdb-f07f-41ad-903e-879ee2d5d80f.png)

Reply:
oops, I missed the sonarlint warning... The solution seems to be `/ {2,}/`, which means _two or more of the character before the curly braces_. regex is a very weird syntax
```suggestion
    .replace(/ {2,}/g, " ") // remove multiple spaces
```